### ISSUE TITLE
[Bug]: Send ETH using Deeplink not working

### ISSUE THREAD

**Issue by seaona** [MetaMask/metamask-mobile#7382]
### Describe the bug

**Problem**: when I try to Send ETH using a Deeplink it does not work for any and I see the error `Something went wrong`.
- I have enough ETH
- This happens to all the networks I've triedd (Sepolia, Goerli, Mainnet)
- It seems it only affects Send ETH, as Send ERC20 seem to work fine

### Expected behavior

Deeplinks should work

### Screenshots

https://github.com/MetaMask/metamask-mobile/assets/54408225/00539d86-cd7a-4b00-9429-bc7559f1f5c0

### Steps to reproduce

1. Create a Send ETH deeplink https://metamask.github.io/metamask-deeplinks/#
2. Scan it
3. See error

### Version

7.7.0 production

### Build type

None

### Device

Pixel 6 + iPhone

### Operating system

iOS, Android

**Comment by sleepytanya** [comment]
Works fine on Android (v7.7.0):


https://github.com/MetaMask/metamask-mobile/assets/104780023/163ad254-e776-4462-8919-52bc60bbe63c

**Comment by seaona** [comment]
thank you for your report @sleepytanya .
I've seen it works for me if I scan your QR code. However, it does not work with the QR code I used in the video. Does the Deeplink work for you if using the QR code I used in the video?

**Comment by seaona** [comment]
[Update] It seems a bug in the Deep link generator website actually. I've noticed that for some reason the link generated was mixed up with a Send ERC20.
Notice there's a transfer keyword in the link, whereas with a normal Send ETH we shouldn't have this param.
This explains why it was not working. I've removed the transfer param and I can proceed with the scan.
I do think we could better handle the Deep link errors, as the user might have no clue on what's going on, but I guess it's a different topic. I close the issue because of this reason
